Lombiq.Npm.Targets.targets

```diff
@@ -29,21 +33,33 @@
   <Target
     Name="NpmDotnetPostclean"
     AfterTargets="Clean"
-    Condition="'$(ExecDotnetPostcleanCommand)' == 'true'">
-    <Exec Command="$(NpmDotnetPostcleanCommand)" />
+    Condition="'$(ExecDotnetPostcleanCommand)' == 'true' AND Exists($(PackageJson))">
+    <Exec Command="$(NpmDotnetPostcleanCommand)"
+          WorkingDirectory="$(NpmTargetsWorkingDirectory)"
+          EnvironmentVariables="@(NpmCommandsEnvironmentVariables)" />
   </Target>
 
   <Target
     Name="NpmDotnetPrebuild"
-    AfterTargets="InitCommonNodeModules;NpmInstall"
+    DependsOnTargets="NpmInstall"
+    AfterTargets="AfterResolveReferences;InitCommonNodeModules"
```

Comment:
I've adjusted this and lines 5-6 according to [this enlightening paragraph](https://docs.microsoft.com/en-us/visualstudio/msbuild/target-build-order?view=vs-2022#determine-the-target-build-order) in the docs. This should better express what's supposed to happen.

I'm not happy with the `InitCommonModules` target here (from GE), but it felt out of scope to change anything about it as part of my issue.

Reply:
Do you mean `InitCommonNodeModules`? And why are you not happy with it?

Reply:
Yes, I meant that.

I'm not happy with it here, in NPM Targets, because it's a concern of GE to run `InitCommonModules` before `NpmDotnetPrebuild`. This project should not have to know that target at all.

Reply:
Let's leave this as it is. This was an initial decision and it seems that it's meant as a generic extension point (the project is about Node, after all).

Reply:
That's why I didn't touch it 😉